datasets/pn_summary/README.md

```diff
@@ -0,0 +1,175 @@
+|---|-----------------|--------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|------------|---------|---------------------------|
+| 1 | 738e296491f ... | مدیریت فروش نفت در دوران تحریم هوشمندانه عمل کرد | به گزارش شانا، علی کاردر امروز (۲۷ دی ماه) در مراسم تودیع محسن قمصری، مدیر سابق امور بین الملل شرکت ملی نفت ایران و معارفه سعید خوشرو، مدیر جدید امور بین الملل این شرکت، گفت: مدیریت امور بین‎الملل به عنوان یکی از تاثیرگذارترین مدیریت‌های شرکت ملی نفت ایران در دوران تحریم‌های ظالمانه غرب علیه کشورمان بسیار هوشمندانه عمل کرد و ما توانستیم به خوبی از عهده تحریم‌ها برآییم (...) | مدیرعامل شرکت ملی نفت، عملکرد مدیریت امور بین‎الملل این شرکت را در دوران تحریم بسیار هوشمندانه خواند و گفت: امور بین الملل در دوران پس از تحریم‎ها نیز می‌تواند نقش بزرگی در تسریع روند توسعه داشته باشد. | Oil-Energy | نفت        | Shana   | https://www.shana.ir/ ... |
+| 2 | 00fa692a17 ...  | سبد محصولات پتروشیمی متنوع می‌شود                 | به گزارش شانا به نقل از شرکت ملی صنایع پتروشیمی، علی‌اصغر گودرزی‌فراهانی با اشاره به اینکه همه طرح‌های در حال اجرای صنعت پتروشیمی براساس پیشرفت فیزیکی و پیش‌بینی زمان راه‌اندازی در قالب طرح‌های جهش دوم و سوم تقسیم‌بندی شده‌اند، اظهار کرد: انتظار داریم که طرح‌های جهش دوم صنعت پتروشیمی که پیشرفت‌های (...)                                                                               | سرپرست مدیریت برنامه‌ریزی و توسعه شرکت ملی صنایع پتروشیمی گفت: تنوع محصولات پتروشیمی ایران با بهره‌برداری از طرح‌های جهش دوم و سوم صنعت پتروشیمی افزایش می‌یابد.                                           | Oil-Energy | پتروشیمی   | Shana   | https://www.shana.ir ...  |
+
+
+### Data Fields
+
+- `id`: (`string`), id of the news.
+- `title`: (`string`), the title of the news.
+- `article`: (`string`) the article of the news.
+- `summary`: (`string`), the summary of the news.
+- `category`: (`classification label`), the category of news in English, with possible values including `Economy`, `Roads-Urban`, `Banking-Insurance`, `Agriculture`, `International`..
+- `categories`: (`string`), the category and sub-category of the news in Persian.
+- `network`: (`classification label`), the name of the news agency, with possible values including `Tahlilbazaar`, `Imna`, `Shana`, `Mehr`, `Irna`..
```

Comment:
Can you add the full list of labels for `category ` and `network` please ?

Reply:
sure!